### PR TITLE
Add JonathanCrd as a code owner for Tables

### PR DESCRIPTION
**Pull request by JonathanCrd**
Adding myself as a code owner for the Tables SDK.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -874,10 +874,10 @@
 # ServiceOwners:                                                   @wonner @yanjungao718
 
 # PRLabel: %Tables
-/sdk/tables/                                                       @christothes
+/sdk/tables/                                                       @christothes @JonathanCrd
 
 # ServiceLabel: %Tables
-# AzureSdkOwners:                                                  @christothes
+# AzureSdkOwners:                                                  @christothes, @JonathanCrd
```
Comment (jsquire) [inline .github/CODEOWNERS:880]:
```suggestion
# AzureSdkOwners:                                                  @christothes @JonathanCrd
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -874,10 +874,10 @@
...
 # ServiceOwners:                                                   @sakash279, @sivethe, @ThomasWeiss, @PaulCheng
```
Comment (jsquire) [inline .github/CODEOWNERS:881]:
```suggestion
# AzureSdkOwners:                                                  @christothes, @JonathanCrd
# ServiceOwners:                                                   @sakash279  @sivethe @ThomasWeiss @PaulCheng
```

**Review by jsquire — COMMENTED** [review]
Not sure how the commas snuck into the existing one, but they shouldn't be there.   If you'd be so kind, let's trim those out.

**Review by jsquire — APPROVED** [review]

**Review by christothes — APPROVED** [review]

**Comment by jsquire** [comment]
Looks like the linter errors are due to the bad formatting previously.   Would you please add the following to the end of [the exceptions list](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS_baseline_errors.txt) since these folks are already grandfathered in with bad data?   

```text
sakash279 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
sivethe is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
ThomasWeiss is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
```

**Comment by JonathanCrd** [comment]
> Looks like the linter errors are due to the bad formatting previously. Would you please add the following to the end of [the exceptions list](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS_baseline_errors.txt) since these folks are already grandfathered in with bad data?
> 
> ```
> sakash279 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
> sivethe is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
> ThomasWeiss is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
> ```

I see, sure thing 👌

**Pull request merged**